src/cloud-api-adaptor/test/e2e/README.md

```diff
@@ -189,6 +189,9 @@ TEST_TEARDOWN=no \
 TEST_PROVISION=no \
 TEST_INSTALL_CAA=no \
 TEST_PROVISION_FILE="${PWD}/skip-provisioning.properties" \
+KBS_TEST=yes \
```

Comment:
Is this the correct variable, or is it azure only? We have the section elsewhere in the doc that says to run
```
export DEPLOY_KBS=yes
```
to run KBS tests - 
We also have `os.Setenv("TEST_KBS", "true")` that is set when ` DEPLOY_KBS=yes`, so was it that you meant to set here as the deployment is a separate step in AZURE that we don't want to conflate with testing?

Reply:
not sure, I too that TEST_KBS variable from the`isTestWithKbs()` source code (test/e2e/common.go). I think there is a difference: if provision=false there won't be a KBS deployment, but you might still want to run the tests against an existing KBS deployment

Reply:
I under the distinction with `DEPLOY_KBS` and `TEST_KBS`, but my main point (which I probably watered down) is that you have documented `KBS_TEST`, not `TEST_KBS`.

Reply:
ohh, eagle eyes. thx 👍